onward/app/weather/controllers/WeatherController.scala

```diff
@@ -32,4 +54,61 @@ class WeatherController(weatherApi: WeatherApi, val controllerComponents: Contro
           Cached(10.minutes)(JsonComponent(views.html.weatherFragments.cityForecast(response)))
         })
     }
+
+  private def readLocationHeaders(implicit request: Request[AnyContent]) = {
+    val CityHeader = "X-GU-GeoCity"
+    val RegionHeader = "X-GU-GeoRegion"
+    val CountryHeader = "X-GU-GeoCountry"
+
+    def getEncodedHeader(key: String)(implicit request: Request[AnyContent]) =
+      request.headers.get(key).map(java.net.URLDecoder.decode(_, "latin1"))
```

Comment:
(Not for PR, but for understanding)

I know this is copy/pasted - any idea why `latin1` over `UTF-8`? My limited understanding is that `UTF-8` can have more than 1 byte per character, and because of that is more robust? 

We also seem to [use UTF-8 most other places](https://github.com/search?q=repo%3Aguardian%2Ffrontend%20UrlDecoder.decode&type=code)?

Reply:
I don't see any reason why we shouldn't use utf-8 👍